audit/audit.go

```diff
@@ -0,0 +1,81 @@
+package audit
+
+import (
+	"os/user"
+
+	"go.mozilla.org/sops/logging"
+
+	"database/sql"
+
+	"fmt"
+
+	_ "github.com/lib/pq"
+	"github.com/sirupsen/logrus"
+)
+
+var log *logrus.Logger
+
+func init() {
+	log = logging.NewLogger("AUDIT")
+}
+
+var auditors []Auditor
+
+func SubmitEvent(event interface{}) {
+	for _, auditor := range auditors {
+		auditor.Handle(event)
+	}
+}
+
+func Register(auditor Auditor) {
+	auditors = append(auditors, auditor)
+}
+
+type Auditor interface {
+	Handle(event interface{})
+}
+
+type DecryptEvent struct {
```

Comment:
should the event struct be tied to `Decrypt` or generic?

Reply:
Same as the database, this is just future proofing. Other events might need other fields.